### ISSUE TITLE
A Rather Cool Design

### ISSUE THREAD

**Issue by solomonshalom** [solomonshalom/The-Abyss#25]
![Anidea](https://github.com/solomonshalom/The-Abyss/assets/71135230/01362c95-9c33-4020-a70c-ad635c783290)
Worked on this cool design, how do I implement it now - hmm 🤔 

BTW, the template for this design can be found [here](https://mega.nz/file/ZjVgXLRa#CWNJ4xlEaM3pUJTCAvXgk-7TMYAw0digCzKv2h1pCcw) and you can view it @ [Pixlr](https://pixlr.com/) 

_View // Edit the file by uploading or dropping it into Pixlr :D_

**Comment by bhavy03** [comment]
hey @solomonshalom i would like to work on this issue

**Comment by solomonshalom** [comment]
> hey @solomonshalom i would like to work on this issue

Lovely! I will assign you :D

**Comment by ashishk1331** [comment]
If I'm allowed to interfere,
Can I try this redesign?
😁

**Comment by solomonshalom** [comment]
> If I'm allowed to interfere, Can I try this redesign? 😁

Sure! since I haven't gotten a response from @bhavy03 - go for it!

**Comment by ashishk1331** [comment]
@solomonshalom acknowledged.

**Comment by ashishk1331** [comment]
@solomonshalom There are severe problems with package.json file,
1. Why engine is set specifically to 16? Is it related deployement platform allows only this specific version of node?
2. Why are the "start" command calling "npm run start" itself? Is this some easter egg?

![image](https://github.com/solomonshalom/The-Abyss/assets/44055846/d50abf3b-6c88-420a-b16f-6d330c91efc8)

**Comment by ashishk1331** [comment]
Also, what should I read first or ask for help to get started? I twiddled with packadge.json to work with NPM. Also, from where can I get the .env.sample or the layout or the keys for the environment.

**Comment by solomonshalom** [comment]
> @solomonshalom There are severe problems with package.json file,
> 
> 1. Why engine is set specifically to 16? Is it related deployement platform allows only this specific version of node?
> 2. Why are the "start" command calling "npm run start" itself? Is this some easter egg?
> 
> ![image](https://user-images.githubusercontent.com/44055846/272173364-d50abf3b-6c88-420a-b16f-6d330c91efc8.png)

Shalom! @ashishk1331,

The `package.json` was set in such a way due to few conflicts that were actively trying to fix because for some reason setting the Node Engine to 18 would potenially break the deployment (not so sure 🤔). On-terms w/ the `start` command, I did change it because `next start` didn't quite work but it must be fixed by now, will test it out!

**Comment by solomonshalom** [comment]
> Also, what should I read first or ask for help to get started? I twiddled with packadge.json to work with NPM. Also, from where can I get the .env.sample or the layout or the keys for the environment.

OH, if you scroll down a bit - you should be able to see the build steps mentioned in the `README.md`. That should do it but if you do need any more help, don't hesitate to ping me!

**Comment by ashishk1331** [comment]
![image](https://github.com/solomonshalom/The-Abyss/assets/44055846/b3e85118-548a-451c-98a7-3dfcc3c20570)
```
ALSO, Don't forget to add the ENV!_
```

How to add this env! What keys do I have to add?

**Comment by rajanarahul93** [comment]
"Please assign the design implementation task and address package.json and environment setup issues to me for resolution and progress."

**Comment by ashishk1331** [comment]
Yeah, now it's getting a bit deranged for me. The repository rules are a bit difficult to follow and I'm unable to understand the code base properly, mainly the package. Json file.

I really liked the POC you put up at top. The design. It was totally doable and it's very eye catching. It would've been fun to build it with all image optimization and responsiveness. 😄

@rajanarahul93 wishes to do it please assign him? @solomonshalom

**Comment by solomonshalom** [comment]
> ![image](https://user-images.githubusercontent.com/44055846/272449990-b3e85118-548a-451c-98a7-3dfcc3c20570.png)
> 
> ```
> ALSO, Don't forget to add the ENV!_
> ```
> 
> How to add this env! What keys do I have to add?

Shalom! @ashishk1331,

Check out the `.env-sample`, it does contain both the instruction and also the environmental variables too!

**Comment by solomonshalom** [comment]
> "Please assign the design implementation task and address package.json and environment setup issues to me for resolution and progress."

Shalom! @rajanarahul93,

OH, sure! Though, most of the environment is already set-up and I don't think there is any additional set-up required other than filling in the ENVs

**Comment by solomonshalom** [comment]
> Yeah, now it's getting a bit deranged for me. The repository rules are a bit difficult to follow and I'm unable to understand the code base properly, mainly the package. Json file.
> 
> I really liked the POC you put up at top. The design. It was totally doable and it's very eye catching. It would've been fun to build it with all image optimization and responsiveness. 😄
> 
> @rajanarahul93 wishes to do it please assign him? @solomonshalom

Shalom! @ashishk1331,

OH, I really apologize for the inconvience! but if you do need any help - please let me know! I will try my best to answer it, Mate!

Regarding the assigning, I will assign you both (Dw, you don't actually have to commit your entire time // effort into it, it's completely non-binding)! :D

**Comment by ashishk1331** [comment]
You're the greatest person I've ever met here. Thank you for pumping me up!🤩

Okay I'll try ma best and ask if I'd any queries. 😤

**Comment by solomonshalom** [comment]
> You're the greatest person I've ever met here. Thank you for pumping me up!🤩
> 
> Okay I'll try ma best and ask if I'd any queries. 😤

That means a lot! Thank You! @ashishk1331

ALSO, sure! I will try my best to tackle the Q's you throw! :D

**Comment by ashishk1331** [comment]
@solomonshalom At last, all things have started working! 😭
Now onto designing. 
Note: After this I would love to work on README to add proper "How to get started" section. Documentation needs repairs.

---


![image](https://github.com/solomonshalom/The-Abyss/assets/44055846/4a59a298-1c62-4c7b-9c44-97fc7df96679)

**Comment by solomonshalom** [comment]
> @solomonshalom At last, all things have started working! 😭 Now onto designing. Note: After this I would love to work on README to add proper "How to get started" section. Documentation needs repairs.
> 
> ![image](https://user-images.githubusercontent.com/44055846/272677163-4a59a298-1c62-4c7b-9c44-97fc7df96679.png)

Shalom! @ashishk1331,

Yay! Excited :D

ALSO, sure! slay it ⚔️ !

**Comment by ashishk1331** [comment]
Okay.

**Comment by ashishk1331** [comment]
@solomonshalom How does this look?
![image](https://github.com/solomonshalom/The-Abyss/assets/44055846/59f19155-890e-4905-81e3-85cbb536a887)

**Comment by solomonshalom** [comment]
> @solomonshalom How does this look? ![image](https://user-images.githubusercontent.com/44055846/272912729-59f19155-890e-4905-81e3-85cbb536a887.png)

Shalom! @ashishk1331,

OoO, that looks great! but I do have a tad bit of suggestions, which you can check out [here](https://excalidraw.com/#room=b6ba7cfd6134ef3d1f6f,cCenC3ODurqLjOd8TiNY8w) :D

P.S. YOUR SO FAST, I am not kidding but Waw!

**Comment by harshkas4na** [comment]
Hey @solomonshalom , I have also some designes of my own in my mind can i work on them or not?

**Comment by solomonshalom** [comment]
> Hey @solomonshalom , I have also some designes of my own in my mind can i work on them or not?

Shalom! @harshkas4na,

Sure! Go for it! Mate :D

**Comment by ashishk1331** [comment]
Are things working here or you all are going for exams now? I myself was busy with mid semester. If things are okay, I'll try  some changes on this Sunday.

**Comment by solomonshalom** [comment]
> Are things working here or you all are going for exams now? I myself was busy with mid semester. If things are okay, I'll try some changes on this Sunday.

Shalom! @ashishk1331,

Yes! but it really depends on @jennifershinshin - she might be quite busy but do pull a PR, she might be able to review and merge it! 🤔

**Comment by jennifershinshin** [comment]
Yes! I'll gladly review the PR and merge if you make one. I have time for that :D

**Comment by ashishk1331** [comment]
Let's try then.
